.github/formula-template.j2

```diff
@@ -0,0 +1,27 @@
+class {{ classname }} < Formula
+  include Language::Python::Virtualenv
+
+  desc "{{ summary|capitalize }}"
+  homepage "{{ homepage }}"
+  url "{{ source_url }}"
+  sha256 "{{ source_checksum }}"
+  # license ""
+  revision 1
+
+  depends_on "rust" => :build
+{%- if package == 'dbt-postgres' %}
```

Comment:
I _think_ Redshift needs the postgres dependency as well

Reply:
fix and tested locally 👍